sdk/src/types/block/protocol/mod.rs

```diff
@@ -1,12 +1,15 @@
 // Copyright 2022 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
+mod work_score;
+
 use alloc::string::String;
 use core::borrow::Borrow;
 
 use crypto::hashes::{blake2b::Blake2b256, Digest};
 use getset::{CopyGetters, Getters};
 use packable::{prefix::StringPrefix, Packable, PackableExt};
+pub use work_score::{WorkScore, WorkScoreParameters};
```

Comment:
use `self` instead and put in same group as `crate`

Reply:
Done